xbmc/utils/CPUInfo.cpp

```diff
@@ -412,6 +412,14 @@ CCPUInfo::CCPUInfo(void)
       }
     }
     fclose(fCPUInfo);
+    //  /proc/cpuinfo is not reliable on some Android platforms
+    //  At least we should get the correct cpu count for multithreaded decoding
+    #if defined(TARGET_ANDROID)
```

Comment:
please don't indent preprocessor instructions. they should expose their full ugliness at the beginning of the lines